csharp/src/Vista.SDK/LocalIdBuilder.cs

```diff
@@ -122,6 +122,27 @@ public LocalId Build()
         return new LocalId(this);
     }
 
+    public LocalIdErrorBuilder Validate(out LocalIdErrorBuilder errorBuilder)
+    {
+        errorBuilder = LocalIdErrorBuilder.Empty;
+        if (PrimaryItem is null || !GmodPath.IsValid(PrimaryItem.Parents, PrimaryItem.Node))
+        {
```

Comment:
Is it possible to end up in the second part of this branch? If there is a nonnull `GmodPath` it should never be invalid.